3ds/WildMidi/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: WinterMute <davem@devkitpro.org>
+
+pkgname=3ds-WildMidi
+pkgver=0.4.2
+pkgrel=1
+pkgdesc='Core softsynth library that can be used with other applications.'
+arch=('any')
+url='https://www.mindwerks.net/projects/wildmidi/'
+license=('GPL')
+options=(!strip libtool staticlibs)
+source=( "https://github.com/Mindwerks/wildmidi/archive/wildmidi-${pkgver}.tar.gz" "wildmidi-0.4.2.patch")
+makedepends=('devkitpro-pkgbuild-helpers')
+sha256sums=(
+'551d43cb6de6019885f933a20b6f3205a92814f50da8b0d8bceac002b9a8109d'
+'5ec6e26bdc8beae5d7723d2c59fad7870154bc71e2809ca4c9b4bd0f46a2a0aa')
+
+build() {
+  cd wildmidi-wildmidi-$pkgver
+
+  source /opt/devkitpro/devkitarm.sh
+  source /opt/devkitpro/3dsvars.sh
+  
+  patch -p1 -i $srcdir/wildmidi-0.4.2.patch
+
+  cmake -DCMAKE_TOOLCHAIN_FILE=/opt/devkitpro/devkitarm.cmake \
+    -DCMAKE_INSTALL_PREFIX=$PORTLIBS_PREFIX \
+    -DCMAKE_C_FLAGS="$CFLAGS $CPPFLAGS" \
+    -DCMAKE_CXX_FLAGS="$CFLAGS -fno-exceptions -fno-rtti" \
+    -DCMAKE_AR="$DEVKITARM/bin/arm-none-eabi-gcc-ar" \
```

Comment:
If I don't have the c_flags, it says undefined reference to getuid and getpwuid when I compile my program.
As for the other 2, I removed them and will push soon. Thank you.

Reply:
It is required, because CMake does not make use of `CPPFLAGS`. (We do [this](https://github.com/EasyRPG/buildscripts/blob/master/shared/common.sh#L83) too)

Reply:
Oh, sorry :).

(Note: The getuid ref error is gone because with the -D__SWITCH__ it will take another compiler path due to this upstream patch: https://github.com/Mindwerks/wildmidi/pull/195)